Dockerfile

```diff
@@ -13,7 +13,7 @@ LABEL maintainer="HMPPS Digital Studio <info@digital.justice.gov.uk>"
 ARG BUILD_NUMBER
 ENV BUILD_NUMBER ${BUILD_NUMBER:-1_0_0}
 
-RUN apk add --no-cache tzdata
+RUN apk add --no-cache tzdata curl
```

Comment:
Should we be adding something to the image that is only used locally for the docker-compose healthcheck? Would it not be better to rework the healthcheck instead?